easybuild/easyconfigs/g/GRASS/GRASS-8.4.0-foss-2023a.eb

```
@@ -95,4 +95,11 @@ sanity_check_paths = {
     'dirs': ['grass%(version_major)s%(version_minor)s']
 }
 
+modextrapaths = {
+    'CPATH': 'grass84/include',
+    'LD_LIBRARY_PATH': 'grass84/lib',
+    'LIBRARY_PATH': 'grass84/lib',
+    'PATH': 'grass84/bin',
```

@pavelToman We can avoid the version hardcoding here:
```suggestion
    'CPATH': 'grass%(version_major)s%(version_minor)s/include',
    'LD_LIBRARY_PATH': 'grass%(version_major)s%(version_minor)s/lib',
    'LIBRARY_PATH': 'grass%(version_major)s%(version_minor)s/lib',
    'PATH': 'grass%(version_major)s%(version_minor)s/bin',
```

Can we add a sanity check command for a tool in `grass84/bin`?

---

seems grass84/bin should not be on the PATH, only some helper bins there.